content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -152,6 +152,40 @@ kubectl delete -k manifests/kustomize/env/dev
 # kubectl delete -k manifests/kustomize/env/gcp
 ```
 
+## Best practices maintaining custom manifests
+
+### Maintain a repo for your manifests
+
+Save the following to a source controlled repo.
+
+File `kustomization.yaml`.
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+# Edit the following to change the deployment to your custom namespace.
+namespace: kubeflow
+# You can add other customizations here using kustomize.
+# Edit ref in the following link to deploy a different version of Kubeflow Pipelines.
+bases:
+- github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref={{% kfp-latest-version %}}
+```
+
+### How to
+* Deploy: `kubectl apply -k $YOUR_REPO`
+* Upgrade:
+    1. (Recommended) backup your data storages for KFP.
+    1. Edit `ref={{% kfp-latest-version %}}` to a version you want to upgrade to.
```

Comment:
Is it possible to use "kustomize edit set" instead for changing the versioning?

Reply:
Not sure, I haven't tried that.

I preferred manually editing it, because we can skip asking users to download `kustomize` tool.